project_report/project_report_latex.py

```diff
@@ -362,17 +363,27 @@ def create_method_section(self):
             with self.doc.create(Subsection('Sequencing', numbering=True)):
                 add_text(self.doc, report_text.get('sequencing'))
 
+            bioinf_reports = {
+                'qc': ('Bioinformatics QC', 'bioinformatics_qc'),
+                'bcbio': ('Bioinformatics Analysis for Human samples', 'bioinformatics_analysis_bcbio'),
+                'variant_calling': ('Bioinformatics Analysis', 'bioinformatics_analysis'),
+                'variant_calling_gatk4': ('Bioinformatics Analysis with GATK4', 'bioinformatics_analysis_gatk4'),
+                'human_variant_calling_gatk4': ('Bioinformatics Analysis with GATK4 for Human samples', 'bioinformatics_analysis_gatk4'),
+                'qc_gatk4': ('Bioinformatics QC with GATK4', 'bioinformatics_qc')
+            }
+
             for bioinfo_analysis_type in self.pi.get_project_analysis_types():
                 bioinfo_version = self.pi.get_bioinformatics_params_for_analysis(bioinfo_analysis_type)
-                if bioinfo_analysis_type == 'qc':
-                    with self.doc.create(Subsection('Bioinformatics QC', numbering=True)):
-                        add_text(self.doc, report_text.get('bioinformatics_qc').format(**bioinfo_version))
-                if bioinfo_analysis_type == 'bcbio':
-                    with self.doc.create(Subsection('Bioinformatics Analysis for Human samples', numbering=True)):
-                        add_text(self.doc, report_text.get('bioinformatics_analysis_bcbio').format(**bioinfo_version))
-                if bioinfo_analysis_type == 'variant_calling':
-                    with self.doc.create(Subsection('Bioinformatics Analysis', numbering=True)):
-                        add_text(self.doc, report_text.get('bioinformatics_analysis').format(**bioinfo_version))
+                subsection, paragraph = bioinf_reports[bioinfo_analysis_type]
+                with self.doc.create(Subsection(subsection, numbering=True)):
+                    add_text(self.doc, report_text[paragraph].format(**bioinfo_version))
+
+            if self.pi.has_rapid_samples():
+                with self.doc.create(Subsection('Rapid Bioinformatics Analysis', numbering=True)):
+                    add_text(self.doc, report_text['bioinformatics_analysis_rapid'])
+                    self.doc.append(' More information on Dragen can be found on the ')
```

Comment:
Not sure why but the report shows no space between the the end of the previous sentence and `More information`